src/h2load.cc

```diff
@@ -551,30 +632,37 @@ void Client::process_timedout_streams() {
     }
   }
 
-  worker->stats.req_timedout += req_inflight;
+  if (worker->current_phase == Phase::MAIN_DURATION) {
```

Comment:
Okay. I have changed. But for one function `on_stream_close`, I didn't because, the `submit_request` function call is at the end of the function. I can change it but then similar lines would have been repeated in the same function. I am not sure whether we want to do that.